src/build_options.rs

```diff
@@ -175,6 +175,10 @@ pub struct BuildOptions {
     #[arg(short = 'f', long, conflicts_with = "interpreter")]
     pub find_interpreter: bool,
 
+    /// Allow prerelease python version
+    #[arg(short, long)]
```

Comment:
```suggestion
    #[arg(long)]
```

Let's drop the short form.